src/main/java/uk/gov/ch/developer/docs/models/user/UserModel.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.ch.developer.docs.models.user;
+
+import uk.gov.companieshouse.session.Session;
+import uk.gov.companieshouse.session.model.SignInInfo;
+
+public class UserModel implements IUserModel {
+
+    private SignInInfo signIn;
+
+    /**
+     * Static method used in thymeleaf to handle null users in a readable fashion.
+     * @param user user who is
+     * @return <code>true</code> if user is not null and is signed in. Otherwise returns
+     * <code>false</code>.
+     */
+    @SuppressWarnings("WeakerAccess")
+    public static boolean isUserSignedIn(final UserModel user) {
+        return user != null && user.isSignedIn();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public boolean isSignedIn() {
+        return signIn != null && signIn.isSignedIn();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public String getEmail() throws IllegalAccessException {
+        throwIfNotSignedIn();
+        return signIn.getUserProfile().getEmail();
+    }
+
+    /**
+     * {@inheritDoc} Returns users email as display id per acceptance criteria.
+     */
+    public String getId() throws IllegalAccessException {
+        return getEmail();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public IUserModel clear() {
+        signIn = null;
+        return this;
+    }
+
+    /**{@inheritDoc}*/
```

Comment:
The inheritdoc annotation does not need squiggly braces, and for consistency should be on a separate line.